packages/appsync-modelgen-plugin/src/__tests__/visitors/gqlv2-regression-tests/appsync-swift-visitor.test.ts

```diff
@@ -17,6 +17,7 @@ const getGQLv2Visitor = (
   generateIndexRules: boolean = true,
   handleListNullabilityTransparently: boolean = true,
   transformerVersion: number = 2,
+  generateModelsForLazyLoadAndCustomSelectionSet: boolean = true,
```

Comment:
I suggest using the false value as this tests are for the regression purpose.

Reply:
updated